lib/cryptozoologist/generator.rb

```diff
@@ -48,12 +48,46 @@ def random
 
     # Generates a string for a street address with a number and street. Only
     # uses animal dictionaries and does not respect config exclusions.
-    # 
+    #
     def street_address
       number = rand(1..9000)
       street = Dictionary.animals.sample
       street = street.split(" ").map! {|word| word.capitalize! }.join(" ")
       "#{number} #{street} #{Dictionary.addresses.sample}"
     end
+
+    def city
+      exclude = Cryptozoologist.configuration.exclude
+      words_formatted = exclude.include?(:words) ? [] : cities_words.map { |word| " #{word.capitalize}" }
+      terminologies = exclude.include?(:terminologies) ? [] : cities_terminologies
+      city_labels =  words_formatted + terminologies
+
+      Cryptozoologist.configure do |config|
+        config.exclude = exclude
+      end
+      "#{one_word_animals.sample.capitalize}#{city_labels.sample}"
+    end
+
+    private
+
+    def one_word_animals
+      Dictionary.animals.select {|animal| animal.split(' ').count == 1}
+    end
+
+    def cities_words
```

Comment:
changing the config during generation is not desired and is a bit of an anti-pattern - i would access the subdictionary directly instead of hacking the config. users shouldn't have their config changing out from underneath them while they're using the gem - even if it isn't impacting them.